src/main/webapp/WEB-INF/tags/instructor/courseStudentDetails/studentInformationTable.tag

```diff
@@ -4,38 +4,55 @@
 <%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn" %>
 <%@ attribute name="studentInfoTable" type="teammates.ui.template.StudentInfoTable" required="true" %>
 <%@ tag import="teammates.common.util.Const" %>
-<div class="well well-plain">
-  <div class="form form-horizontal" id="studentInfomationTable">
-    <div class="form-group">
-      <label class="col-sm-1 control-label">Student Name:</label>
-      <div class="col-sm-11" id="<%=Const.ParamsNames.STUDENT_NAME%>">
-        <p class="form-control-static">${fn:escapeXml(studentInfoTable.name)}</p>
-      </div>
-    </div>
-    <c:if test="${studentInfoTable.hasSection}">
+<div class="panel panel-primary">
+  <div class="panel-body fill-plain">
+    <div class="form form-horizontal" id="studentInfomationTable">
       <div class="form-group">
-        <label class="col-sm-1 control-label">Section Name:</label>
-        <div class="col-sm-11" id="<%= Const.ParamsNames.SECTION_NAME %>">
-          <p class="form-control-static">${fn:escapeXml(studentInfoTable.section)}</p>
+        <div class="col-sm-12">
+          <h4 class="text-bold">Enrollment Details</h4>
         </div>
       </div>
-    </c:if>
-    <div class="form-group">
-      <label class="col-sm-1 control-label">Team Name:</label>
-      <div class="col-sm-11" id="<%= Const.ParamsNames.TEAM_NAME %>">
-        <p class="form-control-static">${fn:escapeXml(studentInfoTable.team)}</p>
+      <div class="form-group student-info-row">
+        <div class="col-xs-4 text-right">
+          <label>Course</label>
+        </div>
+        <div class="col-xs-8" id="<%=Const.ParamsNames.COURSE_ID%>">
+          <a href="${studentInfoTable.courseDetailsLink}">
+            <span>${fn:escapeXml(studentInfoTable.courseId)}</span>
+          </a>
+        </div>
       </div>
-    </div>
-    <div class="form-group">
-      <label class="col-sm-1 control-label">Official Email Address:</label>
-      <div class="col-sm-11" id="<%= Const.ParamsNames.STUDENT_EMAIL %>">
-        <p class="form-control-static">${studentInfoTable.email}</p>
+      <div class="form-group student-info-row">
+        <div class="col-xs-4 text-right">
+          <label>Section Name</label>
```

Comment:
Its possible for courses to not have any sections. So you need to add this condition `<c:if test="${studentInfoTable.hasSection}">` and show the `Section Name` row only if its true

Reply:
Although I have added the condition, I was wondering how can one enroll a student without a section name cause I tried and currently I can't enroll new students or edit details of existing ones by leaving section empty ? I says section can't be empty in both cases.

![image](https://user-images.githubusercontent.com/11634699/41530163-410ce998-72e7-11e8-9a95-e4a06d8c2e4d.png)

![image](https://user-images.githubusercontent.com/11634699/41530261-8d1c0436-72e7-11e8-8862-1afbbaf4b7eb.png)

Reply:
When enrolling students in the course, the section column can be skipped, so that is when they are technically enrolled without a section

Reply:
Ok I see. The section column can be skipped altogether. I was keeping section field empty which was giving an error.